packages/react/src/components/Tooltip/Tooltip.args.ts

```diff
@@ -22,7 +22,7 @@ const dark: TooltipProps = {
 const darkicon: TooltipProps = {
   icon: true,
   label: "This is a tooltip for an icon",
-  theme: "dark", // "light" | "dark" ;
+  iconTheme: "dark", // "light" | "dark" ;
```

Comment:
@Shashika6 why are we changing this to `iconTheme`?

Reply:
The theme is for the tooltip. I added a new prop called iconTheme which is available in twig as well. Since this is dark icon I added the iconTheme.

Reply:
So the twig tooltip has support for applying a different theme to the icon and then the tooltip itself?

Reply:
Oh yea that's right. The icon can be a different color than the tooltip.